src/components/inputs/rich-text-input/rich-text-input.story.js

```diff
@@ -56,14 +59,25 @@ storiesOf('Components|Inputs', module)
     return (
       <Section>
         <Spacings.Stack scale="l">
+          <Value
+            defaultValue={''}
+            render={(value, onChange) => (
+              <TextInput
```

Comment:
Is this extra Input a leftover?

Reply:
no, it makes it easier to test the focus / blur of our input.